packages/components/src/mobile/keyboard-aware-flat-list/index.ios.js

```diff
@@ -48,11 +49,18 @@ export const KeyboardAwareFlatList = ( {
 		onKeyboardWillShow={ () => {
 			this.keyboardWillShowIndicator = true;
 		} }
+		scrollEnabled={ containerStyle ? false : true }
```

Comment:
Same here. It seems a bit weird that we block the scroll and set horizontal to true if we pass the `containerStyle`

Reply:
I answer in comment above